core/src/subgraph/instance_manager.rs

```diff
@@ -303,7 +303,7 @@ impl SubgraphInstanceManager {
                 }
             };
 
-            Ok(())
+            futures03::future::ok(())
```

Comment:
Because there's a lot more `futures` than `futures03` around at this point, I preferred to do it this way to keep the diff under control. But that's a good switch to do in the future.